### DISCUSSION TITLE
tutorials/tls/

### DISCUSSION THREAD

**Discussion by giscus** [go-gost/gost#588] [General]
# tutorials/tls/

https://gost.run/tutorials/tls/

**Comment by zengxiaoyun** [comment]
这个证书是怎么生成的 一定要提供三个证书文件吗?

**Reply by Vanlius** [reply]
GOST在每次运行时自动生成TLS证书，如果未指定任何证书，会使用此证书作为默认证书。

**Comment by asdyyx1** [comment]
我使用acme申请的证书，我把他安装到了`/etc/ssl/`位置，只有`私钥文件: $CERT_INSTALL_DIR/privkey.pem`
和`全链证书: $CERT_INSTALL_DIR/fullchain.pem`，我该怎么设置呢
